yo1k/sandbox/postgresql/readme.md

```diff
@@ -0,0 +1 @@
+Contains examples working with [PostgreSQL](https://www.postgresql.org/).
```

Comment:
Do you really need the `postgresql` package? Consider placing `pgexercises` under `sandbox` and removing `postgresql`.

Reply:
Yes, I do.